CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 5.0.0 (????-??-??)
+
+### Breaking Changes
+
+- Use render props `renderTemplate` instead of a component prop for `template`. This also allows using function components as templates.
```

Comment:
We shouldn't ever say that a function component can be used as a render prop because any hooks in it would be broken.

Reply:
The wording makes it sound like you can do `renderItem={SomeFunctionComponent}`, which is a bug I've seen before elsewhere.